CLI/README.md

```diff
@@ -0,0 +1,83 @@
+![logo](./docs/logo-cli.png)
+
+# axe DevTools CLI Example
+
+This example project demonstrates how axe DevTools CLI is used to detect accessibility issues and generates reports in CSV, HTML, JSON, and XML formats.
```

Comment:
```suggestion
This example project demonstrates how axe DevTools CLI spec mode is used to detect accessibility issues and generate reports in CSV, HTML, JSON, and XML formats.
```

Reply:
generates > we are referring to example project